x/gov/handler.go

```diff
@@ -542,6 +542,7 @@ func (h UpdateElectionRuleHandler) Deliver(ctx weave.Context, db weave.KVStore,
 	}
 	rule.Threshold = msg.Threshold
 	rule.VotingPeriod = msg.VotingPeriod
+	rule.Quorum = msg.Quorum
```

Comment:
Only set this if not-nil.

Consider behavior.
I have a rule with 1/2 threshold and 3/5 quorum.
I want to update it to 3/5 threshold. If I do not mention the quorum, it gets removed?

Or maybe this is right... I keep thinking of PATCH actions and you keep writing POST logic. Maybe I just need to change my verb